lib/p2p/Peer.ts

```diff
@@ -64,8 +64,14 @@ class Peer extends EventEmitter {
   public active = false;
   /** Timer to periodically call getNodes #402 */
   public discoverTimer?: NodeJS.Timer;
-  /** Currencies that we have verified that we can swap with for this peer. */
+  /** Currencies that we have verified that we can swap for this peer. */
```

Comment:
I would say that I swap *with* someone or do you mean that we can do swaps *for* this peer instance?

Reply:
I was thinking in the sense that we have verified that we can swap the currency, and we have verified it for this peer. But I actually like your way better. "Currencies that we have verified we can swap with this peer."